test_files/jsdoc/jsdoc.js

```diff
@@ -137,21 +135,24 @@ class RedundantJSDocShouldBeStripped {
  * @return {void}
  */
 function JSDocWithBadTag() { }
-/** *
+/**
  * For example,
  * \@madeUpTag
-  @type {string} */
+ * @type {string}
+ */
 const c = 'c';
-/***
+/**
  * Don't emit type comments for Polymer behaviors,
  * as this breaks their closure plugin :-(
  *
  * @polymerBehavior
+ * @type {!Object}
  */
 const somePolymerBehavior = {};
-/** *
+/**
  * Don't emit type comments for Polymer behaviors
  * if they are declared via the Polymer function.
-  @type {?} */
+ * @type {?}
+ */
 let Polymer;
-Polymer({ behaviors: ['test'] });
+Polymer({ behaviors: [(/** @type {?} */ ('test'))] });
```

Comment:
Will this break PolymerPass?  Or is fixing that the CL I saw go by...

Reply:
Yes. It turns out that Tobias added a crazy hack to tsickle to work around five failing files in google3. I went ahead and fixed the five files, and now aomarks also fixed PolymerPass, so this is no longer an issue.

Once JSCompiler releases with the fix, we should probably just delete this code.